common/src/main/java/gov/cms/ab2d/common/dto/RoleDTO.java

```diff
@@ -0,0 +1,12 @@
+package gov.cms.ab2d.common.dto;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+public class RoleDTO {
+
+    @NotNull
+    private Long id;
```

Comment:
This means the API caller will have to know the database ID of the role, which requires a database lookup prior to using the API. Let's just use the role name.

Reply:
Yeah I think I'll have to look into using some sort of provider or converter in model mapper to accomplish the lookup